apps/server/src/utilities/pathUtil.ts

```diff
@@ -1,27 +1,21 @@
 import getPath from 'platform-folders';
 import * as path from 'path';
 import { Logger } from '@nestjs/common';
-import { homedir } from 'os';
-import { execFileSync } from 'child_process';
 
 export const getSimbridgeDir = () => {
   try {
     return path.join(getPath('documents'), 'FlyByWireSim', 'Simbridge');
   } catch (e) {
-    Logger.warn('Could not get documents path via WinAPI, trying alternate method', e);
+    Logger.warn(
+      'Could not get documents path via WinAPI, Windows Controlled Folder Access is likely blocking the app. Using AppData as fallback',
```

Comment:
Could we note here a documentation link to either MS or our docs for how to override WCFA if someone wants to do this? Note as a comment on the code, not the log message FYI.

Reply:
Also the console closes by default so only who either disabled the setting or opens it manually will see the error message. Bot command sounds fine to me